src/libs/ReportUtils.js

```diff
@@ -767,6 +767,16 @@ function getIcons(report, personalDetails, defaultIcon = null, isPayer = false)
 
         return [actorIcon];
     }
+    if (isTaskReport(report)) {
+        const ownerEmail = lodashGet(report, 'ownerEmail', '');
```

Comment:
NAB: Avoid using `lodashGet` when accessing an object property at the top level, where we know that the object is defined. `lodashGet` is much slower. Provide a default by short circuiting.

Recommendation:
```js
const ownerEmail = report.ownerEmail || '';
```